builtin/builtin.go

```diff
@@ -154,6 +156,16 @@ var (
 		"option::manifest": map[string]parser.Callable{
 			"platform": codegen.Platform{},
 		},
+		"option::breakpoint": map[string]parser.Callable{
```

Comment:
this is no longer necessary?

Reply:
So far I haven't removed the `option::breakpoint` values... it wasn't clear to me from the other discussion whether we wanted to prevent `breakpoint` from taking run options, or prevent it from taking options at all. If these are unwanted, I can remove them.

Reply:
Okay, I don't see a reason to keep these really, I think they just add complexity as they are redundant with `option::run breakpoint`.